packages/core/src/ui/components/Account/EnableAccountPasswordPrompt/EnableAccountPasswordPrompt.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { Box, Button, Flex, PasswordBox, Text } from '@lace/ui';
+import { Drawer, DrawerNavigation } from '@lace/common';
+
+interface Props {
+  open: boolean;
+  password: string;
+  wasPasswordIncorrect: boolean;
+  translations: {
+    title: string;
+    description: string;
+    passwordPlaceholder: string;
+    wrongPassword: string;
+    cancel: string;
+    confirm: string;
+  };
+  onCancel: () => void;
+  onConfirm: (password: string) => void;
```

Comment:
Same reason as in [my previous suggestion](https://github.com/input-output-hk/lace/pull/901#discussion_r1495848330) for useWalletManager change

```suggestion
  onConfirm: (passphrase: UInt8Array) => void;
```

Reply:
fixed in b80c193c7839186c6059b7703a5764cf39f37085